README.md

```diff
@@ -302,10 +303,13 @@ We build the following packages for the Linux guest environment.
     *  The Python 2 package for Linux daemons and libraries.
 *   `python3-google-compute-engine`
     *  The Python 3 package for Linux daemons and libraries.
+*   `google-compute-engine-oslogin`
+    *  The PAM and NSS modules for [GCE OS Login](https://cloud.google.com/compute/docs/oslogin/)
+*   `gce-disk-expand`
+    *  The on-boot resize scripts for root partition.
 
-The package source for Debian and RPM specs for Enterprise Linux 6 and 7 are
-included in this project. There are also
-[Daisy](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy)
+The package sources (RPM spec files and Debian packaging directories) are also
+included in this project. There are also [Daisy](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy)
 workflows for spinning up GCE VM's to automatically build the packages for
 Debian, Red Hat, and CentOS. See the [README](packaging/README.md) in the
 packaging directory for more details.
```

Comment:
Update the Version Updates section below to include any other specifics for os login etc... just as a point of reference. I believe this current language is actually inaccurate? (or maybe I'm wrong).